src/views/product-tutorials-view/server.ts

```diff
@@ -96,7 +76,10 @@ export async function getCloudTutorialsViewProps() {
 				}),
 				sidebarSections,
 			},
-			product: productData,
+			// Note: should likely remove type casting here,
+			// it's currently needed because "hcp" is not a valid LearnProductName.
+			// Kind of a $TSFixMe.
```

Comment:
Some of the types here were a little rough once I removed the `$TSFixMe` from `stripUndefinedProps`. Figure this is something we can follow up on later, maybe as part of a broader types sweep.

Reply:
Could be a related https://app.asana.com/0/1202097197789424/1202946807363608